tst/RtcpFunctionalityTest.cpp

```diff
@@ -175,6 +175,30 @@ TEST_F(RtcpFunctionalityTest, rembValueGet) {
     EXPECT_EQ(STATUS_RTCP_INPUT_REMB_INVALID, rembValueGet(invalidSSRCLength, SIZEOF(invalidSSRCLength), &maximumBitRate, ssrcList, &ssrcListLen));
 }
 
+static void on_PLI_callback(UINT64 opaque) {
+    *(PBOOL) opaque = TRUE;
+}
+
+TEST_F(RtcpFunctionalityTest, onpli) {
+    initializeEndianness();
+    BYTE rawRtcpPacket[] = {0x81, 0xCE, 0x00, 0x02, 0x00, 0x00, 0x00, 0x01, 0x1D, 0xC8, 0x69, 0x91};
+    RtcpPacket rtcpPacket = {0};
```

Comment:
Would you mind initializing everything first, and then putting a space. Helps readability a bit.

```
    BYTE rawRtcpPacket[] = {0x81, 0xCE, 0x00, 0x02, 0x00, 0x00, 0x00, 0x01, 0x1D, 0xC8, 0x69, 0x91};
    RtcpPacket rtcpPacket = {0};
    KvsPeerConnection kpc = {0};
    KvsRtpTransceiver kvsRtpTransceiver;

   MEMSET(&kvsRtpTransceiver, 0, sizeof(KvsRtpTransceiver));
   ....
```

Reply:
cool